src/pages/dashboard/panels/ImportBreakdownTabs/UnrecognizedTab.tsx

```diff
@@ -16,17 +15,26 @@ import Checkbox from '../../../../components/Input/Checkbox';
 import { markUnrecognizedFile } from '../../../../core/slices/mainpage';
 import UnrecognizedAvdumpedItem from './UnrecognizedAvdumpedItem';
 
+import { useGetFileUnrecognizedQuery, useLazyPostFileAVDumpQuery } from '../../../../core/rtkQuery/fileApi';
+
 function UnrecognizedTab() {
   const dispatch = useDispatch();
 
-  const avdumpKeyExists = useSelector((state: RootState) => !!state.localSettings.AniDb.AVDumpKey);
-  const avdumpList = useSelector((state: RootState) => state.mainpage.avdump);
-  const items = useSelector((state: RootState) => state.mainpage.unrecognizedFiles);
+  const [avdumpList, setAvdumpList] = useState([] as Array<string>);
+
+  const items = useGetFileUnrecognizedQuery({ pageSize: 0 });
   const itemsMarked = useSelector((state: RootState) => state.mainpage.unrecognizedMark);
+  const [avdumpTrigger, avdumpResult] = useLazyPostFileAVDumpQuery();
 
-  const runAvdump = (fileId: number) => dispatch(
-    { type: Events.UTILITIES_AVDUMP, payload: fileId },
-  );
+  const runAvdump = (fileId: number) => {
+    avdumpTrigger(fileId).then(() => {
```

Comment:
I used async/await it looks cleaner, as you don't need the .catch() if you do await, you can do 
```
try {
const avdumpResult = avdumpTrigger(fileId).unwrap()
const tempAvdumpList = avdumpList;
} catch (error) {

}
```

Reply:
done